clang-tools-extra/docs/clang-tidy/index.rst

```diff
@@ -349,6 +349,25 @@ An overview of all the command-line options:
         some-check.SomeOption: 'some value'
       ...
 
+Running Clang-Tidy on CUDA Files
+--------------------------------
+
+:program:`clang-tidy` supports analyzing CUDA source files. To ensure correct
```

Comment:
Worth mentioning that if the user is coming from a GCC + NVCC setup (very likely), they will also need to use the `ArgsToRemove` feature of clang-tidy (or equivalent command-line option) to remove NVCC-specific flags. And use `ExtraArgs` to add the `cuda-path`.